ci/glci/openstack_image.py

```diff
@@ -0,0 +1,70 @@
+import os
+import time
+
+from glci.model import OpenRC
+
+from openstack import connect
+
+class OpenstackImageUploader():
```

Comment:
nit: paranthesis are not required for Python3

Reply:
+1